packages/react-components/src/index.ts

```diff
@@ -1,7 +1,6 @@
 import * as messages from './messages';
 import * as pixels from './pixels';
 import * as text from './text';
-// TODO when LoginLogoutButton is removed, move this out of components into frontend
```

Comment:
I ended up not doing this, and it will not happen, because the auth test utils now have other legitimate use cases inside of `react-components`